src/i18n/locales/en.json

```diff
@@ -27,7 +27,7 @@
   "globalValidationMinValue": "Under minimum value",
   "globalValidationNowOrFuture": "Point in time lies in the past",
   "globalValidationRequired": "Required",
-  "globalValidationSameAs": "Is not identical",
+  "globalValidationSameAs": "The passwords do not match",
```

Comment:
This is too specific, the "global validation" string should fit what the name suggests: same as. It's not specific to passwords here, the password string would need to be set in some password input component.